build/.goreleaser.yml

```diff
@@ -40,7 +40,8 @@ nfpms:
     license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
     
     dependencies:
-      - "newrelic-infra"
+      - newrelic-infra
+      - nrjmx >= 1.5.2
```

Comment:
latest non-jlink release is https://github.com/newrelic/nrjmx/releases/tag/v1.5.3

Reply:
afaik 1.6.x are not published in the linux repos, right?